testsuite/classlibrary/TestSimpleNumber.sc

```diff
@@ -102,4 +102,69 @@ TestSimpleNumber : UnitTest {
 		this.assert(actual == expected, "expected %, got %".format(expected, actual));
 	}
 
+	test_softRound {
+		var val;
+		var testF = {|vals, g, t, s| vals.collect({|num| num.softRound(g, t, s)})};
+
+		val = [-1.3, -1, -0.7, -0.5, -0.3, 0, 0.2, 0.5, 0.7, 1, 1.2, 1.8, 2];
+		this.assertEquals(testF.(val, 1,0,1), [ -1, -1, -1, 0, 0, 0, 0, 1, 1, 1, 1, 2, 2 ], "softRound test 1 failed.");
```

Comment:
keep in mind that `message` is posted even on a successful verbose run. The message should describe the test; these currently read like error messages

Reply:
and again, testing way too many values IMO, it's very hard to tell what's going on in this method

Reply:
Ah, sorry used to unit test libraries that only show the message when it fails. I will modify. I see why people complain about the verbosity of the messages.

Each of the values is testing a very specific edge case where a bug (imo) could creep in. I could split these onto multiple lines, but I'm hesitant to reduce coverage.

Reply:
thank you!